app/views/users/_user_form.html.erb

```diff
@@ -0,0 +1,41 @@
+<style>
```

Comment:
Same comment as before 

Styles should always be defined as external stylesheets in `app/assets`